packages/nextjs/test/integration/pages/api/withSentryAPI/unwrapped/noParams.ts

```diff
@@ -0,0 +1,7 @@
+import { NextApiRequest, NextApiResponse } from 'next';
```

Comment:
I recommend we make a test checking for nested routes like `[...all].ts`

Docs https://nextjs.org/docs/routing/introduction#dynamic-route-segments

Reply:
Done.